content/blog/readxl-1-4-0/index.Rmd

```diff
@@ -0,0 +1,75 @@
+---
+output: hugodown::hugo_document
+
+slug: readxl-1-4-0
+title: readxl 1.4.0
+date: 2022-03-28
+author: Jenny Bryan
+description: >
+    readxl 1.4.0 is a maintenance release with practically no user-facing
+    changes, but extensive change to package internals.
+
+photo:
+  url: https://unsplash.com/photos/tpAyLp9Ro50
+  author: Ryan
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [readxl]
+---
+
+We're pleased to announce the release of [readxl](https://readxl.tidyverse.org) 1.4.0.
+The readxl package makes it easy to get tabular data out of Excel files and into R with code, not mouse clicks.
+It supports both the legacy `.xls` format and the modern XML-based `.xlsx` format.
+readxl is designed to be easy to install (so: no external dependencies) and to cope with many of the less savory features of Excel files created by humans and 3rd party applications.
+
+The easiest way to install the latest version from CRAN is to install the whole tidyverse.
+
+```{r eval = FALSE}
+install.packages("tidyverse")
+```
+
+Alternatively, install just readxl from CRAN:
+
+```{r eval = FALSE}
+install.packages("readxl")
+```
+
+Regardless, you will still need to attach readxl explicitly.
+It is not a core tidyverse package, i.e. readxl is NOT attached via `library(tidyverse)`.
+Instead, do this in your script:
+
+```{r}
+library(readxl)
+```
+
+This release has practically no changes that should be noticeable by the typical user.
+However, internally, there have been extensive updates that set the stage for future user-facing improvements.
+Therefore, this post will be quite short and the main point is to encourage readxl users to kick the tires.
+We set out to upgrade the foundation to support building new features and we'd love to hear about any unintended regressions.
+
+You can see a full list of changes in the [release notes](https://readxl.tidyverse.org/news/index.html).
+
+## Updated libxls
+
+readxl now embeds libxls v1.6.2 (the previous release embedded v1.5.0).
+The libxls project is maintained by Evan Miller and is hosted at <https://github.com/libxls/libxls>, where you can read more in its [release notes](https://github.com/libxls/libxls/releases).
+
+## Switch from Rcpp to cpp11
+
+Thanks to Shelby Bearrows, readxl now uses [cpp11](https://cpp11.r-lib.org) to manage the interface to the embedded RapidXML and libxls libraries.
```

Comment:
> manage the interface to the embedded RapidXML and libxls libraries

I guess that's not quite the right wording, since it's really about a more general C/C++ interface for R objects. Happy to get wording suggestion here.

Reply:
I think you could just delete "to manage the interface to the embedded RapidXML and libxls libraries."